src/Tools/AzureBlobStorage/AzureBlobStorage.csproj

```diff
@@ -0,0 +1,67 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
The way the directory structure is set up and the usage of the Directory.Build.props files, I think this project should be moved to `src/AzureBlobStorage` (move it out of the Tools folder) or nest it into `src/Extensions/AzureBlobStorage` to organize future extensions into a common directory.